docs/contracts/errors.rst

```diff
@@ -0,0 +1,55 @@
+.. index:: ! error
+
+.. _errors:
+
+******
+Errors
+******
+
+Errors in Solidity provide a convenient and gas-efficient way to explain to the
+user why an operation failed. They can be defined inside and outside of contracts.
+
+::
+
+    // SPDX-License-Identifier: GPL-3.0
+    pragma solidity >=0.7.0 <0.9.0;
+
+    /// Insufficient balance for transfer. Needed `required` but only
```

Comment:
```suggestion
    /// Insufficient balance for transfer. Needed  `@param required` but only
```

Reply:
This is on purpose: https://docs.soliditylang.org/en/v0.8.1/natspec-format.html?highlight=natspec#dynamic-expressions